musicbot/bot.py

```diff
@@ -5545,6 +5545,43 @@ async def cmd_config(
                 f"To save the change use `config save {opt.section} {opt.option}`",
```

Comment:
You may need to add `"reset"` to the `valid_options` list, and the check following the `reload` sub-command.